CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## main
 
+* Fix template generated at the project root when generating an ERB component with the `--inline` flag
```

Comment:
```suggestion
* Do not generate template when using `--inline` flag.
```

Reply:
Just so you know, I explained it like this in the changelog because it's actually a fix of the `--inline` flag feature introduced in 1.24.0, not a new feature.